src/ml_tooling/metrics/metric.py

```diff
@@ -0,0 +1,67 @@
+from typing import Union, List
+
+import attr
+import numpy as np
+from sklearn.metrics import get_scorer
+from sklearn.model_selection import cross_val_score
+
+
+@attr.s
+class Metric:
+    metric: str = attr.ib()
+    score: Union[float, int] = attr.ib(default=None)
+    cross_val_scores: np.ndarray = attr.ib(default=None)
+
+    def score_metric(self, estimator, x, y):
+        scoring_func = get_scorer(self.metric)
+        self.score = scoring_func(estimator, x, y)
+
+    @property
+    def std(self):
```

Comment:
I know Sci-Kit learn does this to but you may grossly underestimate variance, http://papers.nips.cc/paper/2468-no-unbiased-estimator-of-the-variance-of-k-fold-cross-validation.pdf . An idea is avalible at https://stats.stackexchange.com/questions/31190/variance-estimates-in-k-fold-cross-validation

Reply:
Agreed - we have an issue #158 on it already, just want to do a 1-1 in this pull requet